appjs/searchbar.js

```diff
@@ -1275,6 +1299,30 @@ function handleclick(value) {
     else if (inp.value.toUpperCase() == "EXCESS-3 CODE CONVERTOR") {
         $("#ex3").slideToggle(); 
     }
+    else if(inp.value.toUpperCase() == "LINEAR REGRESSION CALCULATOR"){
+        document.getElementById('lrcs').click(); 
+    }
+    else if (inp.value.toUpperCase() == "LINEAR REGRESSION CALCULATOR") {
+        $("#lrcs").slideToggle(); 
+    }
+    else if(inp.value.toUpperCase() == "COVARIANCE CALCULATOR"){
+        document.getElementById('covs').click(); 
+    }
+    else if (inp.value.toUpperCase() == "COVARIANCE CALCULATOR") {
+        $("#covs").slideToggle(); 
+    }
+    else if(inp.value.toUpperCase() == "SKEWNESS CALCULATOR"){
+        document.getElementById('skews').click(); 
+    }
+    else if (inp.value.toUpperCase() == "SKEWNESS CALCULATOR") {
+        $("#skews").slideToggle(); 
+    }
+    else if(inp.value.toUpperCase() == "SOLVE FOR X"){
+        document.getElementById('slvxs').click(); 
+    }
+    else if (inp.value.toUpperCase() == "SOLVE FOR X") {
+        $("#slvxs").slideToggle(); 
+    }
```

Comment:
Here also you made duplicate entries

Reply:
That's as per other coded recommendation. We  needed two times for opening and closing. And in each section with and without #.